docs/en/ingest-management/fleet/fleet-settings.asciidoc

```diff
@@ -60,13 +60,14 @@ are updated automatically.
 * `https://abae718c1276457893b1096929e0f557.fleet.eu-west-1.aws.qa.cld.elstc.co:443`
 * `https://[2001:db8::1]:8220`
 
+// lint disable elastic-cloud
 |===
 --
 
 . Under *Outputs*, click the *Edit* icon in the *Action* column to modify
 output settings, as needed.
 +
-NOTE: The Elastic Cloud internal output is locked and cannot be edited. This
+NOTE: The {ecloud} internal output is locked and cannot be edited. This
```

Comment:
Did this "disable" work? It looks like one Elastic Cloud was replaced but the other wasn't.

Reply:
I think I was initially confused because the substitution on L70 worked, but the substitution on the line below didn't. Looking at this again, I'm guessing you manually updated L70 when you added the lint disable.